treeherder/etl/perf_data_adapters.py

```diff
@@ -268,36 +309,37 @@ def adapt_and_load(self, reference_data, job_data, datum):
 
-            # summary series
-            summary_properties = {
-                'suite': _suite,
-                'subtest_signatures': json.dumps(subtest_signatures)
-            }
-            summary_properties.update(reference_data)
-            summary_signature = self._get_series_signature(
-                summary_properties)
-
-            summary_data = self._calculate_summary_data(
-                job_id, result_set_id, push_timestamp, talos_datum["results"])
-
-            obj = self._get_base_perf_obj(_job_guid, _name, _type,
-                                          talos_datum,
-                                          summary_signature,
-                                          summary_properties,
-                                          summary_data)
-            # FIXME: this stuff is pretty big and it's unclear how useful it
-            # might be. consider adding it at some point in the future
-            # (see: https://bugzilla.mozilla.org/show_bug.cgi?id=1142631)
-            # for key in ['test_aux', 'talos_aux', 'results_aux',
-            #             'results_xperf']:
-            #     aux_blob = talos_datum.get(key)
-            #     if aux_blob:
-            #         obj['blob']['metadata'][key] = aux_blob
-
-            validate(obj, self.treeherder_perf_test_schema)
-            self._add_performance_artifact(job_id, summary_signature,
-                                           summary_properties, obj,
-                                           _name, 'summary', summary_data)
+            if subtest_signatures != []:
+                # summary series
+                summary_properties = {
+                    'suite': _suite,
+                    'subtest_signatures': json.dumps(subtest_signatures)
+                }
+                summary_properties.update(reference_data)
+                summary_signature = self._get_series_signature(
+                    summary_properties)
+
+                summary_data = self._calculate_summary_data(
+                    job_id, result_set_id, push_timestamp, talos_datum["results"])
+
+                obj = self._get_base_perf_obj(_job_guid, _name, _type,
+                                              talos_datum,
+                                              summary_signature,
+                                              summary_properties,
+                                              summary_data)
+                # FIXME: this stuff is pretty big and it's unclear how useful it
+                # might be. consider adding it at some point in the future
+                # (see: https://bugzilla.mozilla.org/show_bug.cgi?id=1142631)
+                # for key in ['test_aux', 'talos_aux', 'results_aux',
+                #             'results_xperf']:
+                #     aux_blob = talos_datum.get(key)
+                #     if aux_blob:
+                #         obj['blob']['metadata'][key] = aux_blob
```

Comment:
Let's just remove this comment block-- we've decided what to do with this data. :)